app/models/post.rb

```diff
@@ -1,2 +1,16 @@
 class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :likes, class_name: 'Like'
+  has_many :comments, class_name: 'Comment'
+
+  after_create :update_users_count
+
+  def update_users_count
+    author = User.find(author_id)
+    author.increment(:posts_counter)
+  end
+
+  def five_recent_comments
+    Comment.order('created_at DESC').limit(5)
+  end
```

Comment:
- [ ] Just as discussed above, this method also poses the same problems. Please consider working with this suggestion:

  ```suggestion
  def five_recent_comments
    comments.order('created_at Desc').limit(5)
  end
  ```

Reply:
This issue is well resolved! ✔️